storage/boltz/query_symbols.go

```diff
@@ -340,6 +344,14 @@ type compositeEntitySetSymbol struct {
 	cursorLastF func(tx *bbolt.Tx, key []byte) (FieldType, []byte)
 }
 
```

Comment:
This interface method was missing, which meant we didn't properly build chains when we had more than 1 linked entity in a symbol.